docs/source/manual.rst

```diff
@@ -0,0 +1,140 @@
+Basic User Manual
+=================
+
+This package implements a ``HPGe`` which describes the detector geometry and can be used for:
+
+- visualisation with `pyg4ometry <https://pyg4ometry.readthedocs.io/en/stable/>`_
+- exporting to GDML for Geant4 simulations (again with pyg4ometry),
+- computing detector properties.
+
+Metadata specification
+----------------------
+
+The detector geometry is constructed based on the LEGEND metadata specification `(metadata-docs) <https://github.com/legend-exp/legend-detectors/tree/main/germanium/diodes>`_.
+This consists of a JSON file (or python dictionary) describing the geometry for example:
+
+.. code-block:: python
+
+    metadata = {
+        "name": "B00000B",
+        "type": "bege",
+        "production": {
+            "enrichment": {"val": 0.9, "unc": 0.003},
+            "mass_in_g": 697.0,
+        },
+        "geometry": {
+            "height_in_mm": 29.46,
+            "radius_in_mm": 36.98,
+            "groove": {"depth_in_mm": 2.0, "radius_in_mm": {"outer": 10.5, "inner": 7.5}},
+            "pp_contact": {"radius_in_mm": 7.5, "depth_in_mm": 0},
+            "taper": {
+                "top": {"angle_in_deg": 0.0, "height_in_mm": 0.0},
+                "bottom": {"angle_in_deg": 0.0, "height_in_mm": 0.0},
+            },
+        },
+    }
+
+.. note::
+    Currently bege, icpc, ppc and coax geometries are implemented as well as a few LEGEND detectors with special geometries.
+    Different geometries can be implemented as subclasses deriving from ``legendhpges.base.HPGe``.
+
+The different keys of the dictionary describe the different aspects of the geometry.
+Some are self explanatory, for others:
+
+- "production": gives information on the detector production, we need the "enrichment" to define the detector material,
+- "geometry"  : gives the detector geometry in particular, "groove" and "pp_contact" describe the contacts of detector.
+
+Other fields can be added to describe different geometry features (more details in the legend metadata documentation).
+
+Constructing the HPGe object
+----------------------------
+
+The HPGe object can be constructed from the metadata with:
+
+.. code-block:: python
+
+    from legendhpges import make_hpge
+    import pyg4ometry as pg4
+
+    reg = pg4.geant4.Registry()
+    hpge = make_hpge(metadata, name="det_L")
+
+The metadata can either be passed as a python dictionary or a path to a JSON file.
+
+
+Detector properties
+-------------------
+
+Most detectors are described by a ``G4GenericPolycone`` `(polcyone-docs) <https://pyg4ometry.readthedocs.io/en/stable/autoapi/pyg4ometry/geant4/solid/GenericPolycone/index.html#module-pyg4ometry.geant4.solid.GenericPolycone>`_
```

Comment:
also use an intersphinx link here
```suggestion
Most detectors are described by a ``G4GenericPolycone`` (:class:`pyg4ometry.geant4.solid.GenericPolycone`)
```

that will only work after a rebase onto the just merged #55